docs/ota-client-guide/modules/ROOT/pages/simulate-device-basic.adoc

```diff
@@ -123,7 +123,7 @@ Each directory should contain the following:
 
 * credentials.zip
 * sota_local.toml -- a config file for aktualizr
-* (Optional) A `virtualsec.json` file containing the configuration for one or more secondary ECUs.footnote:[The terms "primary" and "secondary" ECU are used in the Uptane specification. For more information about the difference between primary and secondary ECUs, see our overview of the dev@ota-client::uptane.adoc#_primary_and_secondary_ecus[Uptane framework].]
+* (Optional) A `virtualsec.json` file containing the configuration for one or more secondary ECUs.footnote:[The terms "primary" and "secondary" ECU are used in the Uptane specification. For more information about the difference between primary and secondary ECUs, see our overview of the xref:uptane.adoc#_primary_and_secondary_ecus[Uptane framework\].]
```

Comment:
@tkfu This content is included here https://docs.ota.here.com/getstarted/dev/simulate-device-workstation.html
So a local xref wont work when viewing it from "Get Started"

Reply:
Okay, I've updated. Am I correct that all pages that are included elsewhere have the `:page-partial:` attribute? (Or are in the _partials directory?)

Reply:
Yes, whole pages have the `:page-partial:` attribute and can be anywhere you want. Smaller, content chunks have to live in the partials dir.